src/optimagic/optimizers/tao_optimizers.py

```diff
@@ -102,21 +102,9 @@ def func_tao(tao, x, resid_out):  # noqa: ARG001
     # scaled_gradient_tolerance manually we manually set absolute_gradient_tolerance and
     # or relative_gradient_tolerance to zero once a subset of these two is turned off
     # and scaled_gradient_tolerance is still turned on.
-    default_gatol = (
-        convergence_absolute_gradient_tolerance
-        if convergence_absolute_gradient_tolerance
-        else -1
-    )
-    default_gttol = (
-        convergence_scaled_gradient_tolerance
-        if convergence_scaled_gradient_tolerance
-        else -1
-    )
-    default_grtol = (
-        convergence_relative_gradient_tolerance
-        if convergence_relative_gradient_tolerance
-        else -1
-    )
+    default_gatol = convergence_gtol_abs if convergence_gtol_abs else -1
```

Comment:
Have you thought about truthy / falsy problems here?

Reply:
Since this is unrelated to the PR (it just showed up due to a renaming) I will make an issue.